libs/barista-components/chart/src/tooltip/chart-tooltip.ts

```diff
@@ -96,6 +75,9 @@ export class DtChartTooltip implements OnDestroy {
     private _changeDetectorRef: ChangeDetectorRef,
     private _elementRef: ElementRef<HTMLElement>,
     @Optional()
+    @Inject(TOOLTIP_CONFIG_TOKEN)
+    private customPositioner?: DtChartTooltipConfig,
```

Comment:
The DtChartTooltipConfig may only contain a positioning logic now, but may be extended in the future. Please rename the member variable to something more fitting, like: 

```suggestion
    private chartTooltipConfig?: DtChartTooltipConfig,
```